test/parallel/test-fs-utils-get-dirents.js

```diff
@@ -65,6 +65,31 @@ const filename = 'foo';
     },
     ));
 }
+{
+  // Intentional error in lstat
+  let path = '';
+  if (common.isWindows) {
+    path = 'c:\\dev\\null\\does\\not\\exist';
+  } else {
+    path = '/dev/null/does/not/exist';
+  }
```

Comment:
```suggestion
  const path = resolve('<a verifiable non existent>');
```

The resolve function will add the appropriate path separator. Additionally, you should probably create a tmpdir, and use a nonexistent path within it, as theoretically that path could exist

Reply:
Add `const nonExistentPath = resolve(tmpdir.path, 'does-not-exist');` as a global constant and used it for parameter of function calls